Added jest-cli to the package.json for the Fiber record-tests script.

aickin opened the pull request:
The `./scripts/fiber/record-tests` script currently doesn't work out of the box, because it depends on `jest-cli`, which isn't in package.json. Right now you have to manually `npm install jest-cli` to get the script to run correctly.

This PR just adds `jest-cli` to the package.json so that `record-tests` will work after a usual `npm install`.

Cheers!

gaearon commented:
Mind updating Yarn lockfile too?

Also what's the difference between jest and jest-cli packages? Can/should we use just one or so we really need both?

spicyj commented:
They should be interchangeable I think. Let's just unify one one or the other? I guess `jest` is probably the most preferred name now.

aickin commented:
@gaearon I always forget about yarn! Thanks for reminding me.

I took @spicyj's advice, though, and switched `record-tests` to use `jest` instead of `jest-cli`. I ran the script on my machine and it seemed to work fine; it ran the test suite and updated the timestamps on all three `tests-(passing|failing|passing-except-dev).txt` files. If there's more to do to test `record-tests`, let me know!

spicyj commented:
Thanks!
